apps/block_scout_web/assets/js/pages/survey_banner.js

```diff
@@ -0,0 +1,50 @@
+import { connectElements, createStore } from '../lib/redux_helpers.js'
+
+import $ from 'jquery'
+import omit from 'lodash/omit'
+
+export const initialState = {
+  showBanner: !(localStorage.getItem('showSurveyBanner') === 'false')
```

Comment:
Then it has to be `sessionStorage` as well.

Reply:
Oh, thought I changed that already.  Ok, that is fixed.